app/views/brackets/_bracket.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="table-responsive">
+  <table class="table">
+    <tbody>
+      <tr>
+        <td class="align-middle">
+          <%= render "matchups/matchup", matchup: @bracket.matchup_at(1) %>
+          <%= render "matchups/matchup", matchup: @bracket.matchup_at(2) %>
+          <%= render "matchups/matchup", matchup: @bracket.matchup_at(3) %>
+          <%= render "matchups/matchup", matchup: @bracket.matchup_at(4) %>
```

Comment:
Since these view partials are conventionally named, you can refactor to something like

```erb
          <%= render @bracket.matchup_at(1) %>
```

for each one, or get even more fancy with a loop

```suggestion
          <% (1..4).each do |position| %>
            <%= render @bracket.matchup_at(position) %>
          <% end %>
```

what do you think?